examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<p>Let's drop the GROMACS appendix for now - Once we hammer on the export timing a bit we can put it back in. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Done, and removed from the installation instructions at the top of the notebook.</p>